### PR TITLE
Angular upgrade 1.5.3

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js please review

### CODE REVIEW

**Comment by henrahmagix** [comment]
@Minglee01 This library is still compatible with 1.2, so it shouldn't be pinned to 1.5.3. Instead use a range: `1.2 - 1.5` which means 1.2.0 up to but not including 1.6.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# upcoming
+
+* Allow angular versions between 1.2 - 1.5.
```
Comment (henrahmagix) [inline CHANGELOG.md]: How about

> Upgrade Angular compatibility to include 1.3, 1.4, and 1.5

**Comment by henrahmagix** [comment]
I think this should be a patch upgrade, but unsure

**Comment by Minglee01** [comment]
id say its a MINOR as its not a bug fix.

**Comment by henrahmagix** [comment]
But it's not a new feature on the public API. It's an improvement to the package, but it's not breaking change and it's not a new feature, so it should be a patch upgrade

**Pull request merged**